### DISCUSSION TITLE
Votre API et Discord

### DISCUSSION THREAD

**Discussion by LeMaitre4523** [PapillonApp/papillon-v5#48] [Q&A] [answered]
Bonjour, j'aurai une question par rapport à votre API, pensez-vous la rendre open source ? Et pour Discord, avez-vous un serveur Discord pour ce Projet ?

**Comment by LeMaitre4523** [comment]
Ok j'ai trouvé le lien du Discord

**Comment by sivelswhy** [comment, marked as answer]
Voici l'API, en revanche, je ne suis pas maintainer du project, donc je ne sais pas vraiment quel lien est le meilleur
https://github.com/PapillonApp/pronote-api
https://github.com/PapillonApp/papillon-python

**Reply by LeMaitre4523** [reply]
Je viens de le voir merci beaucoup, c'est le deuxième

**Comment by LeGeek01** [comment]
pronote api est déprecié, papillon python est le mieux et la méthode utilisée par papillon actuellement

**Reply by andronedev** [reply]
Exactement, papillon python est l'api actuellement utilisé 😉
